src/theme/MuiThemeProvider/muiTheme.ts

```diff
@@ -111,24 +111,6 @@ export default createTheme({
     },
   },
   components: {
-    MuiButton: {
-      defaultProps: {
-        variant: 'contained',
```

Comment:
I'd assign defaultProps: { disableRipple: true } here if we don't use this effect

Reply:
We aren't going to use default theme styles because we aren't going to ever use the components directly and may eventually drop MUI